README.md

```diff
@@ -35,7 +35,7 @@ The client could be distributed as wasm library and rust crate. There could also
 ## Testing
 The current `NostrClient` code only uses an in memory local test relay, which must be started before testing.
 
-By now we use the relay at `git@github.com:rodant/bucket.git`. To start the relay locally:
+By now we use the relay at `https://github.com/rodant/bucket`. To start the relay locally:

```

Comment:
What kind of errors do you get? Did you add your ssh key with 'ssh-add <path-to-private-key>'? In any case we can replace the link by the original coracle repository (git@github.com:coracle-social/bucket.git), they merged my PR already.